urns/urns_test.go

```diff
@@ -144,6 +218,12 @@ func TestValidate(t *testing.T) {
 		{"telegram:abcdef", false},
 		{"facebook:12345678901234567", true},
 		{"facebook:abcdef", false},
+
+		// facebook refs can be anything
+		{"facebook:ref:facebookRef", true},
```

Comment:
too late now but wish we were using something else between `ref` and the rest of the path

Reply:
I think its actually URNy to use `:`s in this way but agree its kinda weird